app/services/registry-services.js

```diff
@@ -110,7 +110,7 @@ angular.module('registry-services', ['ngResource'])
         isArray: true,
         transformResponse: function(data/*, headers*/){
           var res = [];
-          console.log(data);
+          //console.log(data);
```

Comment:
Don't be afraid and go ahead and remove this line completely. There's code like this all over the place ;-)